### ISSUE TITLE
[Badges] Failed to install libraries

### ISSUE THREAD

**Issue by fixator10** [TrustyJAID/Trusty-cogs#15]
info.json requires installation of `pybarcode`, but in pypi there is no `pybarcode` package
```
ERROR:red.downloader:Something went wrong when installing the following requirements: pillow, pybarcode
Could not find a version that satisfies the requirement pybarcode (from versions: )
No matching distribution found for pybarcode
```

**Comment by TrustyJAID** [comment]
Looks like pybarcode has been removed from pypi. Give me a bit I can strip the required code or vendor an old version removing the requirement for pybarcode.

**Comment by TrustyJAID** [comment]
Should be fixed in de3ff52. 

Edit: Actually forgot something d9c093a should fix it.

**Comment by fixator10** [comment]
👌 `Cog badges successfully installed.`
